.env.example

```diff
@@ -7,5 +7,7 @@ VITE_APP_SEMESTER=
 VITE_APP_SITE_TITLE=
 VITE_APP_BACKEND_URL=
 
+VITE_APP_SENTRY_DSN=https://01f0882e6aa029a125426e4ad32e6c18@o553498.ingest.us.sentry.io/4507775325437952
+VITE_APP_SENTRY_TRACING=0
 # To upload source maps to Sentry (Optional)
-SENTRY_AUTH_TOKEN=
```

Comment:
I think there is no need for `VITE` prefix here as this env variable is not used in the React app, but in the cli or GitHub CI workflow.